Subject: [PATCH] Revert "input: Adjust priority"

 * This seems to interact poorly with cpusets, just revert it.
   It was improperly merged anyway and didn't become obvious
   until enabling HWC2.

diff --git a/services/inputflinger/InputManager.cpp b/services/inputflinger/InputManager.cpp
index 2ef25f3..6a6547b 100644
--- a/services/inputflinger/InputManager.cpp
+++ b/services/inputflinger/InputManager.cpp
@@ -20,7 +20,6 @@
 
 #include "InputManager.h"
 
-#include <cutils/iosched_policy.h>
 #include <cutils/log.h>
 
 namespace android {
@@ -65,9 +64,6 @@
         mDispatcherThread->requestExit();
         return result;
     }
-
-    android_set_rt_ioprio(mDispatcherThread->getTid(), 1);
-    android_set_rt_ioprio(mReaderThread->getTid(), 1);
 
     return OK;
 }
diff --git a/services/surfaceflinger/DispSync.cpp b/services/surfaceflinger/DispSync.cpp
index 0df3b24..37b6420 100644
--- a/services/surfaceflinger/DispSync.cpp
+++ b/services/surfaceflinger/DispSync.cpp
@@ -22,7 +22,6 @@
 
 #include <math.h>
 
-#include <cutils/iosched_policy.h>
 #include <cutils/log.h>
 
 #include <ui/Fence.h>
@@ -392,8 +391,6 @@
         ALOGE("Couldn't set SCHED_FIFO for DispSyncThread");
     }
 #endif
-
-    android_set_rt_ioprio(mThread->getTid(), 1);
 
     reset();
     beginResync();
diff --git a/services/surfaceflinger/EventThread.cpp b/services/surfaceflinger/EventThread.cpp
index 2dacd87..dd88adb 100644
--- a/services/surfaceflinger/EventThread.cpp
+++ b/services/surfaceflinger/EventThread.cpp
@@ -20,7 +20,6 @@
 #include <sys/types.h>
 
 #include <cutils/compiler.h>
-#include <cutils/iosched_policy.h>
 
 #include <gui/BitTube.h>
 #include <gui/IDisplayEventConnection.h>
@@ -93,7 +92,6 @@
 
 void EventThread::onFirstRef() {
     run("EventThread", PRIORITY_REALTIME);
-    android_set_rt_ioprio(getTid(), 1);
 }
 
 sp<EventThread::Connection> EventThread::createEventConnection() const {
diff --git a/services/surfaceflinger/SurfaceFlinger.cpp b/services/surfaceflinger/SurfaceFlinger.cpp
index 342ef82..6bf565a 100644
--- a/services/surfaceflinger/SurfaceFlinger.cpp
+++ b/services/surfaceflinger/SurfaceFlinger.cpp
@@ -27,7 +27,6 @@
 
 #include <EGL/egl.h>
 
-#include <cutils/iosched_policy.h>
 #include <cutils/log.h>
 #include <cutils/properties.h>
 
@@ -522,7 +521,6 @@
 
     mEventControlThread = new EventControlThread(this);
     mEventControlThread->run("EventControl", PRIORITY_REALTIME);
-    android_set_rt_ioprio(mEventControlThread->getTid(), 1);
 
     // initialize our drawing state
     mDrawingState = mCurrentState;
diff --git a/services/surfaceflinger/SurfaceFlinger_hwc1.cpp b/services/surfaceflinger/SurfaceFlinger_hwc1.cpp
index 67fcd6b..f33b502 100644
--- a/services/surfaceflinger/SurfaceFlinger_hwc1.cpp
+++ b/services/surfaceflinger/SurfaceFlinger_hwc1.cpp
@@ -30,7 +30,6 @@
 
 #include <EGL/egl.h>
 
-#include <cutils/iosched_policy.h>
 #include <cutils/log.h>
 #include <cutils/properties.h>
 
@@ -569,7 +568,6 @@
 
     mEventControlThread = new EventControlThread(this);
     mEventControlThread->run("EventControl", PRIORITY_REALTIME);
-    android_set_rt_ioprio(mEventControlThread->getTid(), 1);
 
     // set a fake vsync period if there is no HWComposer
     if (mHwc->initCheck() != NO_ERROR) {
